hadoop-hdfs-project/hadoop-hdfs/src/test/java/io/hops/security/TestUsersGroups.java

```diff
@@ -835,15 +873,25 @@ public void testConcurrentSetSameOwner() throws Exception {
   }
 
 
+  @Test
```

Comment:
I assume that all test methods test*Cache0 is testing when the cache is disabled and the same for test*Cache1 testing when the cache is enabled. Can you change the methods names to test*CacheEnabled and test*CacheDisabled so it would be easy to understand. 

Also, can you update the description of hops.ug.cache.secs and  hops.ug.cache.size with how to disable the cache in core-default.xml https://github.com/hopshadoop/hops/blob/d1a7981972adafa7d52c7a6879bf38ac17e3ab61/hadoop-common-project/hadoop-common/src/main/resources/core-default.xml#L2424 
https://github.com/hopshadoop/hops/blob/d1a7981972adafa7d52c7a6879bf38ac17e3ab61/hadoop-common-project/hadoop-common/src/main/resources/core-default.xml#L2436

Reply:
done